uktrade/great-cms
# Grtlv 77 ukea csat frontend

This PR is to apply a new Harmonized Customer Satisfaction Tool (HCSAT) to the Export Academy booking success page. 

To test locally:

1) Add the following feature flag to your local config file:
FEATURE_UKEA_CSAT=True
2) visit http://greatcms.trade.great:8020/export-academy/events/
3) Click into any available event
Note: you may need to activate an old event or create a new event in your local database
4) Register for the event
5) When complete you will be redirected to the booking success url and see the start of the csat form.

<img width="897" alt="Screenshot 2024-05-29 at 15 42 01" src="https://github.com/uktrade/great-cms/assets/80459294/bac80ed3-55e2-47f5-b039-49b6733dfa2e">

6) Submitting you satisfaction answer will open the second part of the csat form

<img width="422" alt="Screenshot 2024-05-29 at 15 43 45" src="https://github.com/uktrade/great-cms/assets/80459294/38c6a5c5-e132-456f-b329-2cb049fba638">


### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GRTLV-77

### Reviewing help


### Housekeeping


### Merging

- [x] This PR can be merged by reviewers.
